components/RichText.tsx

```diff
@@ -0,0 +1,95 @@
+import { documentToReactComponents } from '@contentful/rich-text-react-renderer';
+import { BLOCKS, MARKS, INLINES } from '@contentful/rich-text-types';
+import React from 'react';
```

Comment:
**Nickpick:** Keep react as the first import.
**Thought:** Depending on the tooling we should be okay to remove this import, see https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html and perhaps this relevant MR if any config changes are needed for linting: https://git.torchbox.com/sue-ryder/self-help/-/merge_requests/33/diffs#diff-content-dbc0c31823b8f2e4ed04a397722fed33a67f123f